scripts/dataset.py

```diff
@@ -0,0 +1,89 @@
+from datasets import list_datasets, load_dataset
+
+
+wikipedia_dataset = load_dataset('wikipedia', '20200501.en')
+lama_dataset = load_dataset('lama', 'google_re')
+
+num_examples = len(wikipedia_dataset['train']) + len(lama_dataset['train'])
+
+
+def clean_text(text):
```

Comment:
Did you write this function, or is this what EleutherAI uses elsewhere?

I'm sure you could have a lot more sophisticated cleanup, removing HTML tags, other markup, etc

Reply:
Ah, I see that this is just used by the wikipedia dataset. Then I guess we know that say HTML won't be showing up.

In any case, I'd still be curious how did you figure out the rules above to use.